### ISSUE TITLE
Add way to fine-tune the pixel classifier models

### ISSUE THREAD

**Issue by SdeBlank** [RiosGroup/BEHAV3D#36]
Currently you can only apply other previously trained classifiers. It would be nice to have a way to append new ground truth labels to previous classifiers

**Idea:**
- Save the unraveled array of the labels and features for pixels that are labeled in a .npy file in the PixelClassifier folder
- When ticking "fine-tune" in the notebook widget, append newly labeled data to the already available .npy file of previous labeled data without needing the full images and train on the combination
